### PR TITLE
Fixes and improvements for chat command

### PR DESCRIPTION
**Pull request by flexsurfer**
During standup extension development, i fixed a few command bugs and made some improvements to UI

_Note to testers_ Regular sanity checks

### CODE REVIEW

File: src/status_im/ui/screens/chat/styles/animations.cljs [hunk 1]
```diff
@@ -19,6 +19,10 @@
    :bottom           bottom
    :position         :absolute
    :elevation        2
+   :shadow-offset    {:width 0 :height 1}
+   :shadow-radius    12
+   :shadow-opacity   0.12
+   :shadow-color     "rgba(0, 0, 0)"
```
Comment (jeluard) [inline src/status_im/ui/screens/chat/styles/animations.cljs]: white?

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/chat/styles/animations.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/chat/styles/animations.cljs]: yeah anyway i need to fix tests

Comment (flexsurfer) [reply src/status_im/ui/screens/chat/styles/animations.cljs]: it had an alpha channel at first, but then it moved to shadow opacity, i'll fix it

**Review by dmitryn — APPROVED** [review]

**Comment by asemiankevich** [comment]
@flexsurfer for some reason i can't build  ios artefact https://ci.status.im/job/status-react/job/prs/job/ios/view/change-requests/job/PR-7955/  :( cc @jakubgs

**Comment by flexsurfer** [comment]
@asemiankevich let me try to rebase onto latest develop

**Comment by asemiankevich** [comment]
@antdanchenko @Serhy @churik  same here, test is failing

**Comment by asemiankevich** [comment]
@jakubgs there are always random fails in building artifacts here, please take a look.

**Comment by jakubgs** [comment]
>@jakubgs there are always random fails in building artifacts here, please take a look.

Well, if you look at the error it just times out:
```
Cancelling nested steps due to timeout
```
It seems the `Bundle` stage, specifically `gradlew assemblePr`, times out once it hits ~30 minutes, which is far too long. I'm not seeing this happening to other PRs or the nightly, so it's something in this PR. Let me see the diff.

**Comment by jakubgs** [comment]
It seems to me the `Bundle` stage times out because the linux slave runs out of memory:
linux_05_out_of_mem

**Comment by jakubgs** [comment]
Ok, looks like the rebase fixed it:
https://ci.status.im/job/status-react/job/prs/job/android/job/PR-7955/5/
So this must have been a resurgence of an issue that was introduce sometime in between the recent Nix changes.

**Pull request merged**